tsconfig.json

```diff
@@ -1,11 +1,18 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es5",
```

Comment:
I wonder if you could put it back to `esnext` if you set `lib` to `["esnext", "dom"]`?

Reply:
I will try

Reply:
Thanks! I see that this still says "es5". What error do you run into if you change "target" to "esnext"?